app/lib/submission_builder/ty2024/states/nc/documents/d400_schedule_s.rb

```diff
@@ -10,8 +10,9 @@ def document
               build_xml_doc("FormNCD400ScheduleS") do |xml|
                 xml.DedFedAGI do
                   xml.USInterestInc calculated_fields.fetch(:NCD400_S_LINE_18)
-                  add_non_zero_value(xml, :ExmptIncFedRecInd, :NCD400_S_LINE_27)
-                  add_non_zero_value(xml, :TotDedFromFAGI, :NCD400_S_LINE_41)
+                  xml.TaxPortSSRRB calculated_fields.fetch(:NCD400_S_LINE_19)
+                  xml.ExmptIncFedRecInd calculated_fields.fetch(:NCD400_S_LINE_27)
+                  xml.TotDedFromFAGI calculated_fields.fetch(:NCD400_S_LINE_41)
```

Comment:
talking about `add_non_zero_value` for NCD400 
https://codeforamerica.atlassian.net/browse/FYST-471?focusedCommentId=12938